index.js

```diff
@@ -1,109 +1,51 @@
-"use strict";
-
-const fs = require("fs");
-const blockhash = require("blockhash-core");
+const fs = require("fs").promises;
+const { bmvbhash } = require("blockhash-core");
 const { imageFromBuffer, getImageData } = require("@canvas/image");
 const imageType = require("image-type");
 const jpeg = require("jpeg-js");
 
-function hash(filepath, bits, format) {
-  format = format || "hex";
+async function hash(filepath, bits = 8, format = "hex") {
   if (format !== "hex" && format !== "binary")
-    throw new Error("Unsupported format");
+    throw new Error(`Unsupported format "${format}"`);
 
-  bits = bits || 8;
```

Comment:
"should support binary output" test will fail because you converted this line into a default prop which is not the same when you call it like in the test:

```js
await imghash.hash(__dirname + "/files/absolut1", null, "hex");
```

FYI I'm going to improve the API after we merge this PR https://github.com/pwlmaciejewski/imghash/issues/88